components/MouseUncaptureOnLooseFocus.cs

```diff
@@ -0,0 +1,18 @@
+using Godot;
+
+using utilities;
+
+namespace Components;
+
+public partial class MouseUncaptureOnLooseFocus : Node
+{
+    [Export] public bool Active = true;
+
+    public override void _Notification(int notification)
+    {
+		if(Active && notification == MainLoop.NotificationApplicationFocusOut)
+		{
+			InputExtensions.UncaptureMouse();
```

Comment:
The equivalent for recapturing the mouse upon focus is:
```cs
if(notification == MainLoop.NotificationApplicationFocusIn)
{
	InputExtensions.CaptureMouse();
}
```

But there is a problem with it, when I press the windows close application button (the "X") it refocuses and the application does not close.

I will implement the refocus in a followup issue, when we have a proper pause menu #40